lib/order.rb

```diff
@@ -0,0 +1,24 @@
+class Order
+  attr_reader :ordered_items, :total, :menu_dishes
+  def initialize 
+    menu = Menu.new
+    @menu_dishes = menu.menu_dishes
+    @ordered_items = []
+    @total = 0
+  end
+
+  def display_menu
+    @menu_dishes
+  end
+
+  def select_dishes(item_chosen, quantity)
+    @ordered_items.push(item_chosen, quantity)
```

Comment:
Storing these values in an array like this feels a bit risky - it means that throughout your system, you'll be reliant on knowing that the structure of the array should be `[item, quantity, item, quantity]` - if you forgot this order, or if the order in which you were storing information ever changed this could cause problems. Can you think of a more structured way of storing the info which would remove this dependence on a specific order?